en/docs/basics_topics/model_load_save.md

```diff
@@ -109,66 +111,109 @@ Some commonly used `initializer`:
+Before using `load_variables`, we have to prepare a dictionary whose key is the `name` specified when creating `variable` and value is a numpy array. After passing the dictionary to `load_variables`, `load_variables` will find the variable object in the job function based on the key and update the value.
 
 For example:
 
 ```python
-check_point = flow.train.CheckPoint() #constructing object of CheckPoint
-check_point.init() #initialize network parameters 
+@flow.global_function(type="predict")
+def job() -> tp.Numpy:
+    myblob = flow.get_variable("myblob", 
+        shape=(3,3), 
+        initializer=flow.random_normal_initializer()
+        )
+    return myblob
+
+myvardict = {"myblob": np.ones((3,3)).astype(np.float32)}
+flow.load_variables(myvardict)
+print(flow.get_all_variables()["myblob"].numpy())
+```
+
+Although we choose the `random_normal_initializer` initialization method, because `flow.load_variables(myvardict)` updates the value of `myblob`. The final output will be:
 
-#... call job function etc.
+```text
+[[1. 1. 1.]
+ [1. 1. 1.]
+ [1. 1. 1.]]
 ```
 
-### Save Model
+### Model Saving and Loading
+
+We can save or load the model by two methods.
+
+- `oneflow.checkpoint.save` : Responsible for saving the current model to the specified path.
+- `oneflow.checkpoint.get` :  Import a model from the specified path.
+
+The prototype of `save` is as follows which saves the model to the path specified by `path`.
 
-At any stage of training process, we can save the model by calling the `CheckPoint` object's `save` method.
 ```python
-check_point.save('./path_to_save')
+def save(path, var_dict=None)
 ```
-Attention:
 
-* The folder specified by path parameter must be empty, otherwise `save` will raise an error.
+The optional parameter `var_dict` saves the object specified in `var_dict` to the specified path if it is not `None`.
 
-* OneFlow model can be saved in a certain form to the specified path. For more details please refer to the example below.
+The prototype of `get` is as follows which loads the previously saved model specified by the `path`.
 
-* Although OneFlow do not have limitation of `save` frequency, however, if the storage frequency is too high, it will increase the burden of resources such as disk and bandwidth.
+```python
+def get(path)
+```
 
-### Load Model
-We can call the `CheckPoint` object's `load` method to load model from specified path. 
+It will return a dictionary that can be updated into the model using the `load_variables`. 
 
-Here is an example of constructing `CheckPoint` object and loading model from a specified path:
 ```python
-check_point = flow.train.CheckPoint() #constructing object 
-check_point.load("./path_to_model") #load model
+flow.load_variables(flow.checkpoint.get(save_dir))
 ```
 
+Attention：
+
+- The path specified by the `save` should either not exist or empty. Otherwise `save` will report an error (to prevent overwriting the original saved model)
```

Comment:
```suggestion
- The path specified by the `save` should either be empty or not existed. Otherwise `save` will report an error (to prevent overwriting the existed saved model)
```